### PR TITLE
View batch info

### PR DESCRIPTION
**Pull request by VincentJ711**
Finishes our teams ticket. Adjusts routing to make more sense. Ready for graph team to start integrating their components.

### CODE REVIEW

File: src/views/BatchInfoPage/BatchInfoPageContent.tsx [hunk 1]
```diff
@@ -7,6 +7,7 @@ import '../../scss/batch-info-page-content.scss';
 import { IUser } from '../../_reducers/UserReducer';
 import BatchCardLarge from '../../components/BatchCardLarge/BatchCardLarge';
 import { AssociateCard } from '../../components/AssociateCard/AssociateCard';
+import Notifications from '../../components/Notifications/Notifications';
 
 interface IBatchInfoPageContentProps {
```
Comment (tylercodes1) [inline src/views/BatchInfoPage/BatchInfoPageContent.tsx:12]: Not exclusive to batch info, but there are inconsistencies in where we store interfaces. 1/2-2/3 of all interfaces are currently being stored in the ./_reducers path.

File: src/views/HomePage/HomePageContent.tsx [hunk 2]
```diff
@@ -9,6 +9,8 @@ import "firebase/auth";
 import "../../scss/home-page.scss";
 import { Link } from 'react-router-dom';
 import { IUser } from '../../_reducers/UserReducer';
+import RequestTalent from '../../components/RequestTalentModal/RequestTalent';
+import Notifications from '../../components/Notifications/Notifications';
 
 interface IHomePageContentProps {
```
Comment (tylercodes1) [inline src/views/HomePage/HomePageContent.tsx:15]: +1 comment as in BatchInfoPageContent.tsx

**Review by tylercodes1 — APPROVED** [review]
Looks pretty good to me. Just left a comment about interfaces.

File: src/views/BatchInfoPage/BatchInfoPageContent.tsx (hunk 1 above)
Comment (VincentJ711) [reply src/views/BatchInfoPage/BatchInfoPageContent.tsx:12]: yea. if i used an interface that already existed, i left it where it was. if its a new one, i separated it out into a file that should hold most of them. I agree they all should be in one place. I'm pretty sure @nickBarak will consolidate them all, if he hasn't already.

**Review by jwilson717 — COMMENTED** [review]
Everything looks great, but I personally do not like to push code to development that doesn't have full functionality. I know the Graphs team may or may not change the modals, but if you change ...props to ...associate on line 100 of the associateCard.tsx that will fix the empty modal issue. (and changing the tag from h3 to p on line 53 of the associate modal will eliminate another warning). Otherwise I think it all looks great.

**Review by jwilson717 — APPROVED** [review]
All looks good to me. (Thanks for fixing those changes!)

**Pull request merged**